fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -104,7 +100,14 @@ def fit(self, X, y, *, sensitive_features, **kwargs):
         self._post_processed_predictor_by_attribute = threshold_optimization_method(
             sensitive_features, y, scores, self._grid_size, self._flip, self._plot)
 
-    def predict(self, X, *, sensitive_features):
+    def predict(self, X, *, sensitive_features, random_state=None):
+        """
+        :param random_state: set to a constant for reproducibility
+        :type random_state: int
+        """
+        if random_state:
```

Comment:
This is rather less sophisticated than `sklearn` allows, but it is at least a subset

Reply:
Right, that's because I'm not using it with the same sophistication as of now. Didn't want to waste time on that yet